src/extension/panel.ts

```diff
@@ -213,7 +213,7 @@ export class Panel {
 
     public selectByIndex(uri: Uri, runIndex: number, resultIndex: number) {
```

Comment:
The implementation of this method could be moved directly into the API (since that's the only caller), but I don't know if that's how you want to organize it or not.